launch/launch/actions/execute_local.py

```diff
@@ -199,7 +201,14 @@ def __init__(
             self.__output = normalize_to_list_of_substitutions(tmp_output)
         else:
             self.__output = tmp_output
-        self.__output_format = output_format
+
+        # Check if an environment variable is set and use this as a default
+        self.__output_format = os.environ.get(
+            'LAUNCH_OUTPUT_FORMAT', output_format
```

Comment:
There's already a `OVERRIDE_LAUNCH_PROCESS_OUTPUT` being used in this class, can we adjust to be like that one?

```suggestion
            'OVERRIDE_LAUNCH_OUTPUT_FORMAT', output_format
```

Reply:
Good point. I completly changed the implementation so that the envvar overrides all input and not just the default. Thereby the behavior fits the name and the behavior of the other envvar.